xcode/Subconscious/Shared/Components/FirstRun/FirstRunView.swift

```diff
@@ -114,6 +154,8 @@ struct FirstRunView: View {
                 for: FirstRunStep.self
             ) { step in
                 switch step {
+                case .initial:
+                    EmptyView()
```

Comment:
Seems odd. We never want to hit this case, right?

Reply:
On reflection, this was basically a laziness decision. I've remodelled the actions to remove the idea of `.initial` step.